source/plugin/api-versions.rst

```diff
@@ -0,0 +1,81 @@
+============
+API-Versions
+============
+
+This page explains which API versions exists and to which Minecraft version their implementations belong to.
+
++-------------+--------------+----------------+-------------------------------------------+
+| API-Version | Release Date | End of Support | Known Implementations (Minecraft Version) |
++=============+==============+================+===========================================+
+| *Planned*   | ---          | --             | * *SpongeForge (1.13.x)*                  |
+| *8.0.0*     |              |                | * *SpongeVanilla (1.13.x)*                |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.1.0       | 2018-09-06   | Late 2019      | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 7.0.0       | 2018-01-01   | ^              | * SpongeForge (1.12.2)                    |
+|             |              |                | * SpongeVanilla (1.12.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 6.0.0       | 2017-05-02   | 2017-12-31     | * SpongeForge (1.11.2)                    |
+|             |              |                | * SpongeVanilla (1.11.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+| 5.1.0       | 2017-02-25   | 2017-12-31     | * SpongeForge (1.10.2)                    |
+|             |              |                | * SpongeVanilla (1.10.2)                  |
++-------------+--------------+----------------+-------------------------------------------+
+
+It is recommended to always update to the latest stable version of the API or to an LTS version.
+
+.. note::
+
+    You can still release your plugin with an older major API version, but we don't officially support these versions.
```

Comment:
Do we want to explicitly say `You can still release your plugin on Ore`?

Reply:
Might be a good way to reference ore some more from the docs. However, I'm not sure where in the sentence I should add it.

Either

> You can still release your plugin on Ore with an older major API version, ...

Or

> You can still release your plugin with an older major API version on Ore, ...

But it may also add a little bit confusion as ore has nothing to do with the problem at hand.

Reply:
> Do we want to explicitly say `You can still release your plugin on Ore`?

I'd say that we should keep this broad and not make this specifically about Ore - Ore has nothing to do with what this page is about.

"You may still choose to develop and release plugins against versions that have reached the end of support, but the Sponge team will not provide any bug fixes or features that you may encounter when using end of support builds."

Reply:
Fixed